### PR TITLE
Added Overwatch

### LINKED ISSUE #382: Overwatch support : build needed
This special version of wine with Andrew Wessie patches is required to run overwtach : [https://github.com/gamax92/wine-overwatch](https://github.com/gamax92/wine-overwatch)

### PR DESCRIPTION
**Pull request by ImperatorS79**
Overwatch has now a bronze rating on the appdb. Seems that the features inside gamax92 wine version will be implemented in ulterior wine staging version. So I think #382 can be closed.

### CODE REVIEW

File: Applications/Games/Overwatch/Online/script.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+include(["Engines", "Wine", "Verbs", "corefonts"]);
+
+new OnlineInstallerScript()
+    .name("Overwatch")                      
+    .editor("Blizzard")                              
+    .applicationHomepage("http://www.playoverwatch.com/")  
+    .author("ImperatorS79")                     
+    .url("https://eu.battle.net/download/getInstaller?os=win&installer=Overwatch-Setup.exe")                      
+    //The checksum is different each time you download
+    .wineVersion("LATEST_STAGING_VERSION")
```
Comment (madoar) [inline Applications/Games/Overwatch/Online/script.js]: You need to remove the `"` here. `LATEST_STAGING_VERSION` is a constant variable

Comment (ImperatorS79) [reply Applications/Games/Overwatch/Online/script.js]: Done

**Comment by plata** [comment]
@madoar I think @ImperatorS79 cannot merge to master so you should do it when you accept a PR.

File: Applications/Games/Overwatch/application.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+	"name":	"Overwatch",
+	"description": "Overwatch is a team-based multiplayer online first-person shooter video game"
```
Comment (plata) [inline Applications/Games/Overwatch/application.json]: "." missing at the end of the sentence.

**Pull request merged**